config/settings.py

```
@@ -0,0 +1,57 @@
+import os
+from typing import Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)